.ci/functionalTests.groovy

```diff
@@ -107,11 +107,6 @@ pipeline {
       steps {
         deleteDir()
         unstash 'source'
-        retry(3){
```

Comment:
`.ci/scripts/functional-test.sh` is the one in charge to call this script

Reply:
I believe this file can be removed as it's not used anymore: let's do it in another PR